charts/pomerium/templates/_helpers.tpl

```diff
@@ -299,8 +299,23 @@ https
 {{- end -}}
 {{- end -}}
 
+{{/*Creates static configuration yaml */}}
 {{- define "pomerium.config.static" -}}
 address: ":{{ template "pomerium.trafficPort.number" }}"
+grpc_address: ":{{ template "pomerium.trafficPort.number" }}"
+certificate_file: "/pomerium/cert.pem"
+certificate_key_file: "/pomerium/privkey.pem"
+certificate_authority_file: "/pomerium/ca.pem"
```

Comment:
Is this really needed in insecure mode? 
It means that You have to use `--set config.generateTLS=true` flag. 
Related: https://github.com/pomerium/pomerium-helm/pull/106/commits/0b83c46b62c852395c3eb13c1a67f3193085ab7b